content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -47,10 +47,11 @@ A more detailed description of the termination behavior can be found in
 ### Hook handler implementations
 
 Containers can access a hook by implementing and registering a handler for that hook.
-There are two types of hook handlers that can be implemented for Containers:
+There are three types of hook handlers that can be implemented for Containers:
 
 * Exec - Executes a specific command, such as `pre-stop.sh`, inside the cgroups and namespaces of the Container.
 Resources consumed by the command are counted against the Container.
+* TCP - Opens a TCP connecton against a specific port on the Container.
```

Comment:
should we call it tcpSocket and next one httpGet so no new terms are introduced?

Reply:
agreed

Reply:
It doesn't look to me the TCP socket option has been landed.

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L2171-L2174

Reply:
hm, right. https://github.com/kubernetes/kubernetes/blob/cea1d4e20b4a7886d8ff65f34c6d4f95efcb4742/pkg/kubelet/lifecycle/handlers.go#L59-L83 

Maybe this can say that the handler can be specified, but will not work.

Reply:
Is this still haven’t been landed yet?